versioned_docs/version-22.04/graph-views/map-web-create-standard-map.md

```diff
@@ -18,7 +18,22 @@ The display of the **Add a Map** button in the **Map** page means that you are a
 3. Click **Add** to confirm the map's creation.
 
 The new map opens directly in the editor interface.
-Now you can customize your map using the MAP editor features.
+
+## Create a map from an existing container
+
+Use containers when you want to create multiple levels of your map. Each level represents a container.
+- In the view mode, each container becomes a map with its own status.
+- In the editor mode, each level becomes a tab. Then you can navigate between containers using the different tabs.
+
```

Comment:
A titre perso, j'aurais plutôt vu l'explication de ce qu'est un container ici https://docs.centreon.com/docs/graph-views/map-web-create-standard-map/#use-graphical-elements-to-build-your-map
mais je te laisse juge